cypress/integration/participantStatus.spec.js

```diff
@@ -9,21 +29,26 @@ describe('Participants status suite', () => {
     cy.contains('button', 'View My Candidates').click();
   };
 
-  const interviewParticipantForm = (siteId) => {
+  const interviewParticipantForm = () => {
     // let it select today as the date
-    cy.get('input[name=ContactedDate]').clear().type('2022/01/13');
+    // Ensure dates selected are within the phase range of Sacred Macaw
+    cy.get('input[name=ContactedDate]').clear().type('2023/01/13');
     cy.contains('button', 'Submit').click();
   };
 
   const hireParticipantForm = (siteId) => {
     // both dates
-    cy.get('input[name=DateHired]').clear().type('2022/01/14');
-    cy.get('input[name=StartDate]').clear().type('2022/01/15');
+    // Ensure dates selected are within the phase range of Sacred Macaw
+    cy.get('input[name=DateHired]').clear().type('2023/01/14');
+    cy.get('input[name=StartDate]').clear().type('2023/01/15');
```

Comment:
Not the biggest fan of hardcoding the dates, even if that's what the existing test did. Perhaps this could be extracted from the DB or test data CSV beforehand, in case test phases change down the line?

Reply:
I've updated to use the csv data, select the phase I want, and im adding a few weeks to the phase.startDate to ensure the hire occurs within that phase.